htmlbasic/bootstrap2.html/homework2-1.html

```diff
@@ -0,0 +1,172 @@
+  <h1>Alexa's Cafe Menu Friday</h1>
+  <p>We have a selection of American dishes to enjoy!</p>
+
+<div id="myCarousel" class="carousel slide" data-ride="carousel">
+  <!-- Indicators -->
+  <ol class="carousel-indicators">
+    <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+    <li data-target="#myCarousel" data-slide-to="1"></li>
+    <li data-target="#myCarousel" data-slide-to="2"></li>
+  </ol>
+
+  <!-- Wrapper for slides -->
+  <div class="carousel-inner">
+    <div class="item active">
+      <img src="https://media-cdn.tripadvisor.com/media/photo-s/07/a9/fa/d6/cafe-arcadia.jpg" alt="Our Home">
+    </div>
+
+    <div class="item">
+      <img src="https://i1.cdn.stylehub.io/56868/b72189dc8f8f492e250e363a03f51ebf/x/o/sectorlanding-cafes-contact2.jpg" alt="Our Pride">
+    </div>
+
+    <div class="item">
+      <img src="https://media-cdn.tripadvisor.com/media/photo-s/12/b6/9b/12/the-idea-of-84-cafe-is.jpg" alt="Our Life">
+    </div>
+  </div>
+
+  <!-- Left and right controls -->
+  <a class="left carousel-control" href="#myCarousel" data-slide="prev">
+    <span class="glyphicon glyphicon-chevron-left"></span>
+    <span class="sr-only">Previous</span>
+  </a>
+  <a class="right carousel-control" href="#myCarousel" data-slide="next">
+    <span class="glyphicon glyphicon-chevron-right"></span>
+    <span class="sr-only">Next</span>
+  </a>
+</div>
+
+<div class="container">
+  <div class="row">
+
+    <div class="col-sm-4">
+      <h3>Starter</h3>
+<img src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcQA-f6Mv5xswRTX9LUSkQ5GxR9znN4ZqX1l2oOFk4oC92ICZMbF" data-toggle="modal" data-target="#menuItem1"> </img>
+<!--modal-->
+<div class="modal fade" id="menuItem1" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="menuitem1">Mini Corn Dogs!</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <p> All American hot dogs wrapped in a buttery deep fried blanket of yum.
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+
+      </div>
+    </div>
+  </div>
+</div>
+
+
+      <p>Other Starters this week:</p>
+      <p>Nachos, Buffalo Wings, Blooming Onion, Mozz Sticks, Queso Dip, Goat Cheese Salad</p>
+    </div>
+
+
+
+    <div class="col-sm-4">
+      <h3>Plate</h3>
+
+<img src= "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRZDCu_Uh5n-b3bczndy-yNDkLAdSuIr2i-eSP2yphpUPCSORQR" data-toggle="modal" data-target="#menuItem2"> </img>
+     <!--modal-->
+<div class="modal fade" id="menuItem2" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
```

Comment:
This part has the interesting affect of disappearing when hovering over. Not sure why but can also be due to the many errors.